lib/fizzy/capi.cpp

```diff
@@ -110,13 +110,17 @@ inline fizzy::ExecutionResult unwrap(const FizzyExecutionResult& result) noexcep
         return unwrap(result.value);
 }
 
-inline auto unwrap(FizzyExternalFn func, void* context) noexcept
+inline fizzy::ExecuteFunction unwrap(FizzyExternalFn c_function, void* c_context)
 {
-    return [func, context](fizzy::Instance& instance, const fizzy::Value* args,
-               int depth) noexcept -> fizzy::ExecutionResult {
-        const auto result = func(context, wrap(&instance), wrap(args), depth);
-        return unwrap(result);
-    };
+    auto context = std::make_any<std::pair<FizzyExternalFn, void*>>(c_function, c_context);
+
+    static constexpr fizzy::ExecuteFunctionPtr function =
+        [](std::any cntx, fizzy::Instance& instance, const fizzy::Value* args, int depth) noexcept {
```

Comment:
```suggestion
        [](std::any host_ctx, fizzy::Instance& instance, const fizzy::Value* args, int depth) noexcept {
```

I propose to explicitly named these params "host context" to differentiate from "execute context" introduced later.